public/views.py

```diff
@@ -168,4 +168,20 @@ def keys_json(request):
     to_json = json.dumps(data, ensure_ascii=False)
     return HttpResponse(to_json, content_type='application/json')
 
+@cache_control(max_age=307)
+def security(request):
+
+    # FIXME: we may want to sort by firstname,lastname
+    developer_keys = DeveloperKey.objects.select_related(
+            'owner').filter(owner__isnull=False)
+
+    # FIXME this is probably not the way we want to do things for now.
+    SECURITY_TEAM = {'anthraxx', 'rgacogne', 'shibumi', 'jelle', 'sangy', 'Foxboron'}
```

Comment:
Yeah, we should probably create a new group, like developers, trustedusers where these users belong :)

Reply:
We have these fixtures here https://github.com/archlinux/archweb/blob/master/main/fixtures/groups.json which should become data migrations later.

Reply:
Are all of these people on the email in the first place :x?